README.md

```diff
@@ -43,26 +41,25 @@ comes with everything you need to start your research, including:
 
 ## What does Flax look like?
 
-We provide here two examples using the Flax API: a simple multi-layer perceptron and a CNN. To learn more about the `Module` abstraction, please check our [docs](https://flax.readthedocs.io/).
+We provide three examples using the Flax API: a simple multi-layer perceptron, a CNN and an auto-encoder. 
+
+To learn more about the `Module` abstraction, please check our [docs](https://flax.readthedocs.io/), or visit our
+[patterns](https://flax.readthedocs.io/en/latest/patterns/flax_patterns.html) page for additional concrete demonstrations of best practices.
```

Comment:
what about linking to
https://github.com/google/flax/blob/master/docs/notebooks/linen_intro.ipynb

people seem to have liked that notebook.

Reply:
good point